integreat_cms/cms/views/contacts/contact_list_view.py

```diff
@@ -66,5 +75,16 @@ def get(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
                 "contacts": contact_chunk,
                 "archived_count": archived_count,
                 "current_menu_item": "contacts",
+                "search_query": query,
             },
         )
+
+    def post(self, request: HttpRequest, *args: Any, **kwargs: Any) -> HttpResponse:
+        r"""
+        Apply the query and filter the rendered contacts
+        :param request: The current request
+        :param \*args: The supplied arguments
+        :param \**kwargs: The supplied keyword arguments
+        :return: The rendered template response
+        """
+        return self.get(request, *args, **kwargs, search_data=request.POST)
```

Comment:
This looks dangerous to me and might lead to an indefinite recursion. I think because of `search_data` this is not going to happen, but it still doesn't look ideal to me. I would be interested to know what @jonbulz thinks

Reply:
I don't think I see the infinite recursion, could you elaborate? It is already implemented this way for other models, i.e. POIs. However, I do agree that it is not ideal, mainly because in the "get" method of a view I would always expect a GET request. To have a POST request end up in the "get" method is a bit unintuitive to me. IMO the search_input_form should also submit a GET request, but changing that now would be a huge pain.

Reply:
will be solved as #3558

Reply:
> I don't think I see the infinite recursion, could you elaborate?

I think I was wrong :)